src/quoi-feur.ts

```diff
@@ -0,0 +1,38 @@
+import type { Message } from 'discord.js';
+import { config } from './config';
+
+const quoiDetector = new RegExp(/\b\s*[qQ][uU][oO][iI]\s*[.,!?]*\s*$/i);
+const ONE_MINUTE = 60000;
+
+const reactWithFeur = async (message: Message) => {
+  await message.react('🇫');
+  await message.react('🇪');
+  await message.react('🇺');
+  await message.react('🇷');
+};
+
+const reactWithCoubeh = async (message: Message) => {
+  await message.react('🇨');
+  await message.react('🇴');
+  await message.react('🇺');
+  await message.react('🇧');
+  await message.react('🇪');
+  await message.react('🇭');
+  await message.react('🔇');
+
+  message.member?.roles.add(config.discord.mutedRoleId);
```

Comment:
Using the role instead of using the redis, I don't know about that.
What happens if the role is not there ?
We try to make every feature "pure", aka not depending on external setup.
(like the Lobby audio Channel that can be created with /createLobby)

An easy solution would be to use the key-value here, otherwise I think you should add a way to create this role automatically (with a command?)

(on another note It make me think that we could have some sort of generic tag system that would use discord roles internally to track the state of some users, I made a discussion #35 )

Reply:
I've add role creation on start-up, let me know if you prefer this solution